src/GraphCanvas.tsx

```diff
@@ -135,7 +141,7 @@ export const GraphCanvas: FC<GraphCanvasProps & { ref?: Ref<GraphCanvasRef> }> =
             legacy
             linear
             flat
-            gl={GL_DEFAULTS}
+            gl={{ glOptions, ...GL_DEFAULTS }}
```

Comment:
Might be better in a useMemo, generally don't want to generate a new object in the render block.

```javascript
const gl = useMemo(() => {
  return {
    ...GL_DEFAULTS,
    ...(glOptions || {})
  }
}), [glOptions]);
```

The above is a little janky may need to add glOptions with and empty object to defaultProps.